views.py

```diff
@@ -347,3 +349,35 @@ def post(self, request):
                 'Unidentified Error',
                 status=status.HTTP_500_INTERNAL_SERVER_ERROR,
             )
+
+
+class CSVImportExport(APIView):
+    """
+    API endpoint that allows importing and exporting csv files
+    """
+
+    def get(self, request, *args, **kwargs):
+        """
+        Returns a blank xlsx file for given model
+        :return: a blank xlsx file for given model
+        """
+
+        model_name = request.GET.get('model')
+        if model_name is None or model_name not in models:
```

Comment:
No need for this `O(n)` approach. You can simply catch the error raised, i.e. `ImproperlyConfigured` and return a response.